docs/eigenda/integrations-guides/dispersal/overview.md

```diff
@@ -0,0 +1,216 @@
+
+The source of truth for the Disperser API spec is [disperser.proto](https://github.com/Layr-Labs/eigenda/blob/8ec570b8c2b266fad20ea0af14f0f5d84906c39c/api/proto/disperser/disperser.proto), adjusted to the current release. The goal of this document is to explain this spec at a higher level.
+
+<!-- TODO: Update to point to master, not a specific commit -->
+
+Eigen Labs hosts one disperser endpoint for each EigenDA network. These endpoints are documented in respective network pages [indexed here](../../networks/README.md).
+
+The EigenDA Disperser exposes 4 endpoints:
+
+1. `DisperseBlob()`
+2. `DisperseBlobAuthenticated()`
+3. `GetBlobStatus()`
+4. `RetrieveBlob()`
+
+These endpoints enable the blob lifecycle, from enqueuing blobs for dispersal to waiting for their dispersal finalization and finally to retrieving blobs from the EigenDA network. The following flowchart describes how move blobs through this lifecycle with respect to these endpoints:
+
+```mermaid
+graph TD;
+    A[Blob Ready] --> |"DisperseBlob()"| B[Blob Queued for Dispersal];
+    A --> |"DisperseBlobAuthenticated()"| B;
+    B -->|"GetBlobStatus()" != finalized| B;
+    B -->|"GetBlobStatus()" == finalized| C[Blob Dispersed and Finalized];
+    C -->|"RetrieveBlob()"| C;
+    C -->|15 days elapses since dispersal finalization| D[Blob Expired];
+```
+
+The Disperser offers an asynchrounous API for dispersing blobs, where clients should poll the `GetBlobStatus()` endpoint with the dispersal request ID they received from calling one of the two disperse endpoints until the disperser reports the blob as successfully dispersed and finalized.
+
+## Endpoints
+
+### Dispersal Endpoints
+
+Blobs can be dispersed through `DisperseBlobAuthenticated()` or `DispersedBlob()` endpoints. The `DispersedBlob()` is used for easy testing of essential functionality in testing environments, sometimes with IP-based whitelists for increased throughput. All production traffic will eventually be required to use the DisperseBlobAuthenticated endpoint. Both endpoints accept a [DisperseBlobRequest](https://github.com/Layr-Labs/eigenda/blob/4d19b0597d1151a402e328430a0ff5345a831fb6/api/proto/disperser/disperser.proto#L72) message, with the following fields:
+
+#### DisperseBlobRequest
+
+| Field Name              | Type     | Description |
+|-------------------------|----------|-------------|
+| `data`                  | []byte   | The data to be dispersed. The blob dispersed must conform to the [Blob Serialization Requirements](blob-serialization-requirements.md) which ensure that the blob's KZG commitment may be representative of the original data that was sent to the disperser. |
```

Comment:
I agree that this callout should be prominent. 

Is it possible to put a warning box within the table or otherwise emphasize this text?

Reply:
I'll bold the text for now. I removed the warning box from before but can add it back in tomorrow.